### ISSUE TITLE
node/no-unpublished-bin false positive

### ISSUE THREAD

**Issue by aladdin-add** [mysticatea/eslint-plugin-node#146]
+ eslint version: v5.11.1
+ eslint-plugin-node: 8.0.0

+ package.json: 

```js
 "bin": "bin/cli.js",
  "files": [
    "LICENSE",
    "README.md",
    "lib",
    "bin/*"
  ],
```

output: 
![image](https://user-images.githubusercontent.com/13050025/50547717-22267580-0c7a-11e9-8ed5-3a9bc0dc79c5.png)

it should **not** report this error, since the "bin/cli.js" has been added in `package.json`. 

I've made a repro: https://github.com/Aladdin-ADD/eslint-plugin-node-issue146

**Comment by mysticatea** [comment]
Thank you for this report. This has been fixed in 0225b02d0f3f8dbf2d2d52577455bf9817e925ca...0593c67576c86b2cbf4990db38d1cc4707b2f309.
